qiskit/quantum_info/operators/measures.py

```diff
@@ -350,6 +350,94 @@ def cvx_bmat(mat_r, mat_i):
     return sol
 
 
+def unitary_diamond_distance(channel1: Operator, channel2: Operator) -> float:
```

Comment:
Rename these inputs to something like `op1` and `op2`. We don't want to call them `channel` if the input shouldn't be a QuantumChannel.

Reply:
Sounds good!